### ISSUE TITLE
WPE can't see the content on the second monitor

### ISSUE THREAD

**Issue by ZeDarkAdam** [pieces-app/support#228]
### Software

Desktop Application, Pieces OS

### Operating System

Windows

### Your Pieces OS Version

9.0.4

### Early Access Program

- [ ] Yes, this is related to an Early Access Program feature.

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

copilot receives information only from the main monitor

**Comment by mark-at-pieces** [comment]
hey @ZeDarkAdam sorry your experiencing this issue,

right now how the live context works is that we can only get context from your focused window, would you mind attempt to focus on windows from your second application to double check this?

**Comment by ZeDarkAdam** [comment]
Yes, the context from the window that has focus is retrieved, but only if that window is on the main monitor/screen

https://github.com/pieces-app/support/assets/80891624/f5c63e06-af5a-4321-9514-a1811653c396

**Comment by mark-at-pieces** [comment]
very interesting, I can see your on windows.

I was able to test this out on my mac, and didnt have any issues but I would not be surprised at all if this is not working properly on a second monitor on your windows machine.

I will checkin with a few other on the team to see if they can reproduce this issue so we can get to the root issue.

Looks like windows11? or are you on windows 10?

**Comment by ZeDarkAdam** [comment]
Windows 11 23H2 (Build 22631.3593)
